_components/icons.md

```diff
@@ -0,0 +1,30 @@
+---
+permalink: /components/icons/
+layout: styleguide
+type: component
+title: Icons
+category: UI components
+lead: Icons communicate meaning in a graphical user interface.
+---
+
+The Design System uses a subset of [Font Awesome](https://fontawesome.com/) icons on the project. They are available for use in `src/img`. If you need additional icons, use [Font Awesome](https://fontawesome.com/) in your project.
+
+<div class="usa-accordion-bordered">
+  <button class="usa-button-unstyled usa-accordion-button"
+      aria-expanded="true" aria-controls="icons-docs">
+    Documentation
+  </button>
+  <div id="icons-docs" aria-hidden="false" class="usa-accordion-content">
+    <h4 class="usa-heading">Implementation</h4>
+    <p>Use the Design System icons available in <code>src/img</code> or <a href="https://fontawesome.com/how-to-use/on-the-web/setup/getting-started?using=web-fonts-with-css">add Font Awesome to your project</a>. This project uses icons in two ways:</p>
+    <ul>
+      <li>SVGs with PNG fallbacks</li>
+      <li>CSS background images</li>
+    </ul>
+    <h4 class="usa-heading">Accessibility</h4>
+    <p>Provide descriptive text when the icons are meaningful or have an action associated with it. Do not provide additional text when the icon is decorative.</p>
```

Comment:
Grammatical nit but I think something like this reads a little better:

`Provide descriptive text when an icon is meaningful, or has an action associated with it.`

Reply:
Changed to a singular icon in 86e0293 although the added comma is unnecessary.

Reply:
If this looks good, could you approve the PR? Thanks!